### DISCUSSION TITLE
stuck during installation: self signed cert - cannot find rhn-ssl-tool

### DISCUSSION THREAD

**Discussion by ipanmeri** [uyuni-project/uyuni#8544] [Q&A] [answered]
Hello,
I am totally new to uyuni (or servermanager) and prepared an openSuse leap server for uyuni installation.
Following the official documentation.

During download/install of the packages I get:

`Retrieving: concurrent-1.3.4-277.278.8.uyuni3.noarch.rpm ................................................................................[error (2.0 KiB/s)]
Download (curl) error for 'https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Server-POOL-x86_64-Media1/noarch/concurrent-1.3.4-277.278.8.uyuni3.noarch.rpm':
Error code: Curl error 60
Error message: SSL certificate problem: self signed certificate in certificate chain
`

I try to follow the official documentation:
https://www.uyuni-project.org/uyuni-docs/en/uyuni/administration/ssl-certs-selfsigned.html
but am stuck already with 

`rhn-ssl-tool: command not found`

As far as I can see this rhn-ssl-tool is nothing openSuse provides, correct?
I am very surprised, that official Uyuni documentation would refer/use a tool not by openSuse ...

What am I missing here?
Thanks for your patience.
Regards

**Comment by mcalmer** [comment]
"https://download.opensuse.org" - here you still try to install Uyuni and download a package from openSUSE infrastructure. This is not the point which is discussed in the docs you point to. In the docs we talk about when a client registered at Uyuni has problems to download packages from Uyuni.

This "self signed certificate in chain" error sounds to me like a wrong configured mirror. Here we cannot really help. Maybe you can talk to the openSUSE infrastructure people? Check the zypper.log which target server was really used. "download.opensuse.org" redirect to a mirror and does not serve the content directly.

**Comment by ipanmeri** [comment]
Hi and thanks for your comment.
Sorry, forgot to put this link in the original post, this is the doc I follow when trying to install uyuni:
https://www.uyuni-project.org/uyuni-docs/en/uyuni/installation-and-upgrade/install-uyuni.html
and there I find
`zypper ar https://download.opensuse.org/$repo uyuni-server-stable
`

Yes, I still try to install Uyuni, and trying to find an answer to the certificate error I found the link to that documentation, but I guess I am jumping around a bit to easily.

So, if this is not the correct place to ask about installation error, where can I turn to?

Thanks a lot and kind regards

**Reply by ipanmeri** [reply]
Is this a good forum to start with openSuse questions?
https://forums.opensuse.org/

Just signed up ... and confirmation link does not work (url not found)
Or maybe, it's just not my day ... :-(

Regards


[Edit]
Ok, shortened the url down until some menu came up. Email confirmed.

**Reply by mcalmer** [reply]
Another option would be to reach out to the openSUSE Heroes https://lists.opensuse.org/archives/list/heroes@lists.opensuse.org/

**Comment by ipanmeri** [comment, marked as answer]
Shame on me, it was a firewall issue.
I hade access allowed to *.opensuse.org ... but was not aware that it points to ftp.gwdg.de. Added that to the firewall rule and wooosh, installation went through.

Thanks a lot for your time and patience.
Have a nice day.
